use-symfony-reproducer.md

```diff
@@ -0,0 +1,166 @@
+cd symfony
+```
+
+Now we’re going to add the original Symfony repo as an _upstream_ remote. Why?
+So we can easily pull in the latest Symfony updates and stay fresh!
+
+```bash
+git remote add upstream git@github.com:symfony/symfony.git
+```
+
+Let’s grab the latest changes from Symfony (you’ll want to run this command
+whenever Symfony gets an update, so it’s good to get used to it).
+
+```bash
+git fetch upstream
+```
+
+## Use your Reproducer to Fix the Bug
+
+In Part 1, you picked the Symfony version for your reproducer. Now it’s time
+to match that version in our forked Symfony repo. Let’s check it out:
+
+```bash
+git checkout 7.1
+```
+
+Boom! You’ve got the right version. Now, let’s make sure we’re up-to-date
+by merging in any new changes:
+
+```bash
+git merge upstream/7.1
+```
+
+Great! Now, we’re going to create a new branch for our bug fix:
+
+```bash
+git checkout -b my-fix
+```
+
+***TIP
+The branch name (`my-fix`) isn't important, but it should be descriptive and
+unique to your fork.
+***
+
+## Linking your Reproducer to your Fork
+
+Let’s link that reproducer you created to your freshly forked Symfony repo.
+Head back to your reproducer directory:
+
+```bash
+cd ~/my-reproducer
+```
+
+Symfony comes with a handy PHP script to link itself to a project. This will
+symlink all your app’s `symfony/*` packages to your local fork. Super neat, right?
+
+```bash
+php ../symfony/link .
+```
+
+## Fixing the Bug
+
+Open your reproducer in your favorite IDE. Normally, messing with files in the 
+`vendor/` directory is a no-no, but in this case, for files in `vendor/symfony/*`,
+you can! Changes you make to these files  are actually being made in your forked
+Symfony repository.
+
+Once you’ve squashed that bug, jump back into your local Symfony fork and take a
+look at your changes:
+
+```bash
+cd ~/symfony
+git status
+```
+
+If everything looks good, follow the Symfony documentation on
+[Creating a Pull Request](https://symfony.com/doc/current/contributing/code/pull_requests.html#step-4-submit-your-pull-request)
+to submit your fix to the Symfony repository!
+
+## Verifying a Proposed Fix
+
+So, you’ve found a bug, and someone from the awesome Symfony community has proposed a fix.
+Now, let’s use your reproducer to check if the fix works.
+
+First, track down the Pull Request (PR) that has the fix. You’ll see something like this
+at the top of the PR:
+
+> {user} wants to merge X commit(s) into symfony:{version} from {user}:{branch-name}
+
+Click `{user}:{branch-name}` to jump to the `{user}`'s fork. Like we did with
+our own fork, click the green _Code_ dropdown and copy the URL to your clipboard.
+
+We’re going to add their fork as a remote:
+
+```bash
+cd ~/symfony
+git remote add {user} <fork-url-from-clipboard>
+git fetch {user}
+```
+
+Now, checkout the branch from the user's fork:
+
+```bash
+git switch -c {user}-{branch-name} {user}/{branch-name}
+```
+
+***TIP
+I like to prefix the branch name with the user’s GitHub handle
```

Comment:
Could we rephrase this "user's GitHub handle"? I got it intuitively, but not sure everyone will get it